projects/igniteui-angular/src/lib/directives/input/input.directive.ts

```diff
@@ -52,7 +52,7 @@ export class IgxInputDirective implements AfterViewInit, OnDestroy {
      */
     @Input('value')
     set value(value: any) {
-        this.nativeElement.value = value;
+        this.nativeElement.value = value ?? '';
```

Comment:
Ah, seems like 8 is using an older that 3.7 typescript that doesn't support the coalesce operator. Will consider our options.